build.gradle

```diff
@@ -23,7 +23,7 @@ ext {
     gutenbergMobileVersion = 'v1.92.0'
     wordPressAztecVersion = 'v1.6.3'
     wordPressFluxCVersion = 'trunk-fcede535112327abf59416b7b544cf526c44fd00'
-    wordPressLoginVersion = '1.0.0'
+    wordPressLoginVersion = '116-9e75c5927871287fccb0617f15f69bb066656570'
```

Comment:
@RenanLukas Just a reminder that this artifact is published to test a [PR](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/116) and shouldn't be merged to WPAndroid's `trunk`. Our dependencies should always be a tag or an artifact published from `trunk` (such as `trunk-{commit-sha}`) before merging WPAndroid PRs.

I am addressing this as part of the code freeze, but wanted to leave this reminder for future PRs. Let me know if you have any questions.